index.bs

```diff
@@ -692,15 +764,14 @@ Registry instances are initially created with the internal slots described in th
 
 <h3 id="loading-operations">Loading Operations</h3>
 
-<h4 id="request-fetch" aoid="RequestFetch">RequestFetch(loader, key)</h4>
+<h4 id="request-fetch" aoid="RequestFetch">RequestFetch(entry)</h4>
```

Comment:
all `Request*()` abstract operations are now using `entry` instead of `key`, and the internal `[[Loader]]` backpointer on every entry.